src/test/ui/annotate-snippet/missing-type.rs

```diff
@@ -0,0 +1,5 @@
+// compile-flags: --error-format human-annotate-rs
+
+pub fn main() {
+    let x: Iter;
+}
```

Comment:
You need to run `./x.py test src/test/ui --stage 1 --bless` for the tests to pass.